epoxy-composeinterop-maverickssample/src/androidTest/java/com/airbnb/epoxy/composeinterop/maverickssample/MultiKeyComposeInteropFragmentTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.airbnb.epoxy.composeinterop.maverickssample
+
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onAllNodesWithText
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MultiKeyComposeInteropFragmentTest {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+    @Test
+    fun testComposableInInteropUsingMutableStateUpdatesEvenWithoutKey() {
+        for (i in 0..10) {
+            waitUntil("withoutInteropKey mutableState: $i, recomposingCount: ${i + 1}")
```

Comment:
The current logic that's using waitUntil waits for a given text to show up but it doesn't guarantee anything since there are no assertions being made. It can recompose 50 times after this for loop and we aren't ensuring that those recompositions aren't happening. Unfortunately, its not verifying what these tests were hoping to verify.

Reply:
Maybe I misinterpreted the logic. Feel free to correct me if you feel otherwise!